articles/api-management/api-management-faq.md

```diff
@@ -141,6 +143,13 @@ If you use Git Credential Manager, or if you're trying to clone a Git repository
 ### Does API Management work with Azure ExpressRoute?
 Yes. API Management works with Azure ExpressRoute.
 
+### Why do we require a dedicated subnet in Resource Manager style VNET for API Management?
+The dedicated subnet requirement for API Management comes from the fact, that it is built on PAAS V1 (Classic) technology. While we can deploy into a Resource Manager VNET (V2 layer), there are consequences to that. The Classic deployment model in Azure are not tightly coupled with Resource Manager model and so if you create a resource in V2 stuff, the V1 doesn't know about it and problems can happen such as API Management trying to use an IP that is already allocated to a NIC (built on V2).
```

Comment:
The dedicated subnet requirement for API Management comes from the fact, that it is built on PAAS V1 (Classic) technology. While we can deploy into a Resource Manager VNET (V2 layer), there are consequences to that. The Classic deployment model in Azure is not tightly coupled with the Resource Manager model and so if you create a resource in V2 model, the V1 resources don't know about it and problems can happen such as API Management trying to use an IP that is already allocated to a NIC (built on V2).